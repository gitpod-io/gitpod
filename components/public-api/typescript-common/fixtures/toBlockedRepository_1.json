{
  "id": 2023,
  "urlRegexp": "*/*",
  "blockUser": false,
  "createdAt": "2023-11-29T18:17:16.194Z",
  "updatedAt": "2023-11-29T18:17:16.194Z"
}
