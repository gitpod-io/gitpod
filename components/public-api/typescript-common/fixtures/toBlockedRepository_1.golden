{
    "result": {
        "id": 2023,
        "urlRegexp": "*/*",
        "blockUser": false,
        "blockFreeUsage": false,
        "creationTime": "2023-11-29T18:17:16.194Z",
        "updateTime": "2023-11-29T18:17:16.194Z"
    },
    "err": ""
}
